src/wallet/rpcdump.cpp

```diff
@@ -819,4 +826,4 @@ UniValue dumpwallet_zcoin(const UniValue& params, bool fHelp)
     dumpParams.push_back(params[0]);
 
     return dumpwallet(dumpParams, false);
-}
+}
```

Comment:
Also I found that there are bitcoin tests in 0.14 where `walletdb.ListPubCoin` fails due to the physical absence of wallet file. I fixed this in core14 branch by adding a check if the wallet file name is empty

Reply:
Ok great. I will add your changes to this PR anyway